### PR TITLE
feature: pr + issue template and commitlint

### PR DESCRIPTION
**Pull request by Wabri**
- Codeowner template
- issue template for bugs
- pull request template
- workflow:
  - remove gitguardian (is running on gitguardian itself)
  - add commit lint

### CODE REVIEW

File: .github/workflows/workflows/commit-lint.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: latest
+          only-new-issues: false
+          args: --timeout 2m --config .golangci.yaml
```
Comment (davideimola) [inline .github/workflows/workflows/commit-lint.yml]: This part may be useful only in case the project is using Go. It is not a good position to me

**Review by davideimola — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+- [ ] Test A
+- [ ] Test B
+
+**Test Configuration**:
+* Firmware version:
+* Hardware:
+* Toolchain:
+* IGN Version:
+
+## Checklis
```
Comment (nicpuppa) [inline .github/pull_request_template.md:31]:
```suggestion
## Checklist
```

Comment (nicpuppa) [inline .github/pull_request_template.md]:
```suggestion
## Checklist
```

**Review by nicpuppa — CHANGES_REQUESTED** [review]

File: .github/workflows/workflows/commit-lint.yml (hunk 1 above)
Comment (Wabri) [reply .github/workflows/workflows/commit-lint.yml]: Ops 👍

**Comment by Wabri** [comment]
@davideimola remove golang linting 😬

**Comment by davideimola** [comment]
@Wabri what do you think to add an issue template for the Feature request?

**Comment by Wabri** [comment]
Yes, you are right!

**Pull request closed without merging**